sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Triggers/EventHubTriggerBindingStrategy.cs

```diff
@@ -49,6 +49,7 @@ public EventData[] BindMultiple(EventHubTriggerInput value, ValueBindingContext
         public Dictionary<string, Type> GetBindingContract(bool isSingleDispatch = true)
         {
             var contract = new Dictionary<string, Type>(StringComparer.OrdinalIgnoreCase);
+            contract.Add("TriggerPartitionContext", typeof(TriggerPartitionContext));
```

Comment:
Since TriggerPartitionContext derives from PartitionContext, could we just update the existing binding of "PartitionContext" to return a TriggerPartitionContext? Is there any downside to doing so?

Reply:
Because the parameter type in the function is `PartitionContext`, I believe we'd be downcast and the new member wouldn't be visible without an explicit cast in the Function body.

Reply:
Nevermind, I guess this won't work as it would break anyone binding to the original type.